binlog_driver/reader/offset.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package reader
+
+import (
+	"github.com/Shopify/sarama"
+	"github.com/ngaut/log"
+	pb "github.com/pingcap/tidb-tools/binlog_proto/go-binlog"
+)
+
+// KafkaSeeker implements Kafka Seeker
+type KafkaSeeker struct {
```

Comment:
we can move this file to pkg

Reply:
it's hightly relate to binlog, i think better to keep it in this dir

Reply:
wormhole and tidb-binlog all use this file, shall we use a normal pkg in this project in future?

Reply:
sure